src/pydex_app/sra_routes.py

```diff
@@ -30,17 +29,77 @@ def get_asset_pairs():
     information required to trade them.
     http://sra-spec.s3-website-us-east-1.amazonaws.com/#operation/getAssetPairs
     """
-    current_app.logger.error("not implemented")
-    raise NotImplementedError()
+    current_app.logger.info("############ GETTING ASSET PAIRS")
+    network_id = NetworkId(int(request.args.get("networkId"))).value
+    assert network_id == current_app.config["PYDEX_NETWORK_ID"], \
+        f"networkId={network_id} not supported"
+    page = int(request.args.get("page", current_app.config["OB_DEFAULT_PAGE"]))
+    per_page = int(request.args.get(
+        "per_page", current_app.config["OB_DEFAULT_PER_PAGE"]))
+    asset_data_a = request.args.get("assetDataA")
+    asset_data_b = request.args.get("assetDataB")
+    asset_pairs, asset_pairs_count = Orderbook.get_asset_pairs(
+        asset_data_a=asset_data_a,
+        asset_data_b=asset_data_b,
+        page=page,
+        per_page=per_page
+    )
+    res = {
+        "total": asset_pairs_count,
+        "perPage": per_page,
+        "page": page,
+        "records": asset_pairs
+    }
+    assert_valid(res, "/relayerApiAssetDataPairsResponseSchema")
+    return current_app.response_class(
+        response=json.dumps(res),
+        status=200,
+        mimetype='application/json'
+    )
 
 
 @sra.route("/v2/orders", methods=["GET"])
+@cross_origin()
 def get_orders():
     """GET Orders endpoint retrieves a list of orders given query parameters.
     http://sra-spec.s3-website-us-east-1.amazonaws.com/#operation/getOrders
     """
-    current_app.logger.error("not implemented")
-    raise NotImplementedError()
+    current_app.logger.info("############ GETTING ORDERS")
+    network_id = NetworkId(int(request.args.get("networkId"))).value
+    assert network_id == current_app.config["PYDEX_NETWORK_ID"], \
+        f"networkId={network_id} not supported"
+    page = int(request.args.get("page", current_app.config["OB_DEFAULT_PAGE"]))
+    per_page = int(request.args.get(
+        "per_page", current_app.config["OB_DEFAULT_PER_PAGE"]))
+    orders, orders_count = Orderbook.get_orders(
+        maker_asset_proxy_id=request.args.get("makerAssetProxyId"),
+        taker_asset_proxy_id=request.args.get("takerAssetProxyId"),
+        maker_asset_address=request.args.get("makerAssetAddress"),
+        taker_asset_address=request.args.get("takerAssetAddress"),
+        exchange_address=request.args.get("exchangeAddress"),
+        sender_address=request.args.get("takerAssetAddress"),
+        maker_asset_data=request.args.get("makerAssetData") or request.args.get(
+            "traderAssetData"),
```

Comment:
What is `traderAssetData` and why is it being used both for maker and taker?

Reply:
http://sra-spec.s3-website-us-east-1.amazonaws.com/#operation/getOrders